flaskr/blueprints/auth.py

```diff
@@ -43,7 +43,7 @@ def login():
     return render_template('auth/login.html', loginform=form)
 
 
-@auth.route('/signup', methods=['GET', 'POST'])
+@auth.route('/sign_up', methods=['GET', 'POST'])
```

Comment:
Would rather have this as `signup` instead of `sign_up` since it's a publicly visible route.